vm/src/stdlib/sys.rs

```diff
@@ -442,6 +442,52 @@ mod sys {
         .into_struct_sequence(vm))
     }
 
+    #[pyfunction]
+    fn __unraisablehook__(
+        unraisable: PyRef<UnraisableHookArgs>,
+        vm: &VirtualMachine,
+    ) -> PyResult<()> {
+        use super::PyStderr;
+
+        let stderr = PyStderr(vm);
+        if !vm.is_none(&unraisable.object) {
+            if !vm.is_none(&unraisable.err_msg) {
+                write!(stderr, "{}: ", unraisable.err_msg.repr(vm)?);
```

Comment:
Oh here also should be `str`
https://github.com/python/cpython/blob/main/Python/errors.c#L1292-L1299
```c
if (err_msg != NULL && err_msg != Py_None) {
    if (PyFile_WriteObject(err_msg, file, Py_PRINT_RAW) < 0) {
    	return -1;
    }
    if (PyFile_WriteString(": ", file) < 0) {
    	return -1;
    }
}
```